app/AppCycle.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * bool $active
+ * date $deadline
+ * int $user_id
+ * string $name
+ */
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class AppCycle extends Model
+{
+    protected $casts = [
+        'active' => 'boolean',
+    ];
+
+    protected $fillable = [
+        'deadline',
+        'name',
+        'user_id',
+    ];
+
+    /**
+     * Scopes
+     */
+
+    public function scopeCurrent($query)
+    {
+        return $query->where('active', true)->first();
+    }
```

Comment:
this isn't a scope
```php
public static function current()
{
    return static::where('active', true)->first();
}
```
it also depends on there always being only one active, which doesn't seem to be guaranteed anywhere (it'd be best to guarantee at model-level)